Dockerfile

```diff
@@ -7,4 +7,7 @@ COPY yarn.lock ./yarn.lock
 RUN yarn install --production
 COPY . .
 
+RUN chown -R node /src
```

Comment:
What does this line do?

Reply:
changes the owner of the /src folder to the node user, it's just to ensure that the permissions are all correct for the executing user.

Reply:
Cool I think it looks good